client/lib/olark/index.js

```diff
@@ -148,10 +156,32 @@ const olark = {
 		olarkEvents.on( 'api.chat.onOperatorsAway', olarkActions.setOperatorsAway );
 		olarkEvents.on( 'api.chat.onOperatorsAvailable', olarkActions.setOperatorsAvailable );
 
+		olarkExpandedEvents.forEach( ( event ) => olarkEvents.on( event, () => {
```

Comment:
Maybe better still we could just do away with the name and make the action itself the focus of attention:

```js
syncStoreWithExpandedState() {
	const isOlarkExpanded = !! document.querySelector( '.olrk-state-expanded' );
	if ( isOlarkExpanded !== olarkStore.get().isOlarkExpanded ) {
		olarkActions.setExpanded( isOlarkExpanded );
	}
}

hookExpansionEventToStoreSync( eventName ) {
	olarkEvents.on( eventName, syncStoreWithExpandedState );
}

…

olarkExpandedEvents.forEach( hookExpansionEventToStoreSync );
```